packages/components/Select/styles.js

```diff
@@ -47,7 +47,7 @@ export const OptionsList = styled.div`
   position: relative;
   display: flex;
   flex-direction: column;
-  width: 100%;
+  width: 100%;                                                                                            
```

Comment:
pls, could u delete this redundant spaces?

Reply:
Wow